lab-css-recipes-clone-master/starter-code/french-onion-soup-starter-code/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>French Onion Soup Recipe</title>
+  <link rel="stylesheet" type="text/css" href="css/main.css"> 
+  <!-- Google fonts gives us access to the Open Sans Font -->
+  <!-- Google fonts is also awesome! Make sure to check it out -->
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,600" rel="stylesheet">
+</head>
+<body>
+  <header class = "title">
+      <h1>French Onion Soup</h1>
+    </header>
+    <section class ="recipe">
+      <p>
+       We have been trying French onion soup in restaurants for years and my family
+      and friends agree none can compare to my recipe for taste and simplicity of
+      preparation.
+      </p>
+      <img src = "images/recipe-info.png" width="300px">
+      <hr>
+    </section>
+    <section class="ingredients">
+      <h3>Ingredients</h3>
+        <ul class = "ingredientsUl">
+          <li>- 1/2 cup unsalted butter</li>
+          <li>- 2 tablespoons olive oil</li>
+          <li>- 4 cups sliced onions</li>
+          <li>- 4 (10.5 ounce) cans beef broth</li>
+          <li>- 2 tablespoons dry sherry (optional)</li>
+          <li>- 1 teaspoon dried thyme</li>
+          <li>- salt and pepper to taste</li>
+          <li>- 4 slices French bread</li>
+          <li>- 2 slices Swiss cheese, diced</li>
+          <li>- 1/4 cup grated Parmesan cheese</li>
+        </ul>
+      <img src = "images/cooking-info.png" width="300px">
+      <hr>
```

Comment:
Also, you can use  [border-bottom](https://www.w3schools.com/cssref/pr_border-bottom.asp)